### DISCUSSION TITLE
Add rel="nofollow" to Curator Column with link

### DISCUSSION THREAD

**Discussion by dave2084** [awcodes/filament-curator#545] [Q&A]
I have a curator column that has the url property set to make it a link, however I am using temporarySignedRoutes for the URLs so I don't want bots (specifically google) to index them, however if I add rel="nofollow" to extraAttributes it is added to the div inside the anchor.  

How can I add it to the anchor?

```
CuratorColumn::make('cover')
    ->height(120)
    ->grow(false)
    ->defaultImageUrl(asset('/images/pdf.svg'))
    ->url(fn (Document $record): string => $record->getSignedDownloadUrl())
    ->extraAttributes([ 
        'rel' => 'nofollow',
    ])
    ->openUrlInNewTab(),
```

In the browser I get the following:
```
<a href="#" target="_blank" class="flex w-full disabled:pointer-events-none justify-start text-start">
    <div class="curator-column px-4 py-3 !p-0" rel="nofollow">
        <div style="" class="">
            <img src="/curator/media/..." alt="..." style="" class="">
        </div>
    </div>
</a>
```

**Comment by awcodes** [comment]
I don't think you can add attributes to URL's on image columns. The CuratorColumn is just an extension of filament's ImageColumn.

**Reply by dave2084** [reply]
Thanks.  I'll have to figure out some other way to do it.
